ql/src/test/results/clientnegative/udf_assert_true2.q.out

```diff
@@ -36,7 +36,7 @@ STAGE PLANS:
                   Limit
                     Number of rows: 2
                     Select Operator
-                      expressions: (1 + assert_true((_col6 < 2))) (type: int)
+                      expressions: (1 + UDFToInteger(assert_true((_col6 < 2)))) (type: int)
```

Comment:
I haven't done research on this, but I would guess this was caused by the fact that in the previous version, it didn't go through CBO, whereas CBO must have added this slight performance regression.  I can file a separate Jira on this.

Reply:
On a second, I guess we don't have to follow up on this. I assume using `assert_true` with arithmetic operations would cause the same conversion even when lateral views are not present in the query.